fabfile.py

```diff
@@ -556,7 +554,6 @@ def build_virtualenv():
                     venv_config.set(sec, option, val)
                 with open(vefile, 'w') as f:
                     venv_config.write(f)
-    run('rm /tmp/distribute* || echo "ok"')  # clean after myself
```

Comment:
With the `--no-setuptools`, does python not create mess in the `/tmp` folder?

Reply:
distribute was the fork of setuptools several years ago, that was since merged back into setuptools. We don't use distrubute anymore.

Reply:
With or without the --no-setuptools, it doesn't create this /tmp/distribute files. This was some old code when we used distribute instead of setuptools on this project.